index.js

```diff
@@ -4,15 +4,51 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = 'red'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travelerImageObj = document.querySelector("div#profile").children[0]
+travelerImageObj.src = traveler.photo
+travelerImageObj.alt = 'dog'
+
+const travelerName = document.querySelector("div#profile").children[1]
+travelerName.innerText = traveler.name
```

Comment:
👍  Good! You can also use `textContent` instead of `innerText`. The main difference is that `textContent` doesn't consider the CSS whereas `innerText` does. [Take a look at this stackoverflow post about it](https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext)